support/gathering-cluster-data.adoc

```diff
@@ -40,8 +40,10 @@ include::modules/gathering-data-specific-features.adoc[leveloffset=+2]
 
 * link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy]
 
+ifdef::openshift-rosa,openshift-dedicated[]
```

Comment:
I couldn't find the preview page to check out if this was properly displaying - am not familiar with the doc set and didn't find a page "gathering data about your cluster" but I might be confused. If so, please ignore.